src/yamux/yamux.lua

```diff
@@ -141,8 +141,26 @@ local yamux_fields = {
     next_frame_ack = ProtoField.framenum("yamux.next_frame", "Next frame (ACK)", base.NONE, frametype.ACK),
 }
 
+-- Set up expert information
+local yamux_expert_info = {
+    -- errors
+    unknown_packet_type = ProtoExpert.new("yamux.expert.unknown_packet_type", "Unknown packet type", expert.group.UNDECODED, expert.severity.ERROR),
+    malformed_packet_type = ProtoExpert.new("yamux.expert.malformed_packet_type", "Malformed packet type", expert.group.MALFORMED, expert.severity.WARN),
+    open_default_stream = ProtoExpert.new("yamux.expert.open_default_stream", "Default opened stream cannot be opened", expert.group.MALFORMED, expert.severity.WARN),
+    data_before_start = ProtoExpert.new("yamux.expert.data_before_start", "Stream packets were detected without SYN flag", expert.group.SEQUENCE, expert.severity.ERROR),
+
+    -- dissector helpful
+    corrupted_frame = ProtoExpert.new("yamux.expert.debug.corrupted_frame", "Frame is corrupted", expert.group.DEBUG, expert.severity.WARN),
+    undecoded_direction = ProtoExpert.new("yamux.expert.debug.undecoded_direction", "Cannot detect client-server direction. Addresses are equal", expert.group.DEBUG, expert.severity.NOTE),
+
+    -- for user debugging
+    client_packet = ProtoExpert.new("yamux.expert.packet_type.client", "Current packet is generated by client", expert.group.DEBUG, expert.severity.COMMENT),
+    server_packet = ProtoExpert.new("yamux.expert.packet_type.server", "Current packet is generated by server", expert.group.DEBUG, expert.severity.COMMENT),
```

Comment:
This isn't a blocking comment, but I'm wondering, Section "7.4.1.3. Group" of <https://www.wireshark.org/docs/wsug_html_chunked/ChAdvExpert.html> states that the the `Debug` group should not be seen in release versions of Wireshark. However, I am seeing debug comments in the expert info for Yamux traffic.

Should the code be doing something different here so that these messages do not appear in the standard expert info output?

Reply:
Thanks, fixed.

I used the section "11.3.6.1. ProtoExpert.new(abbr, text, group, severity)" of [official documentation](https://www.wireshark.org/docs/wsdg_html_chunked/lua_module_Proto.html) while implementing expert info.
This page contains all required Lua names for groups & severities.